rosdep/python.yaml

```diff
@@ -4082,15 +4088,10 @@ python3-flake8:
   ubuntu: [python3-flake8]
 python3-gitlab:
   debian:
-    '*': [python3-gitlab]
-    jessie:
-      pip:
-        packages: [python-gitlab]
+    buster: [python3-gitlab]
   ubuntu:
-    '*': [python3-gitlab]
-    trusty:
-      pip:
-        packages: [python-gitlab]
+    bionic: [python3-gitlab]
+    cosmic: [python3-gitlab]
```

Comment:
Same here, we need rule for all active ubuntu distributions, with this change there is no rule for `trusty`, `xenial` and `artful`.
Can you please apply the same logic as suggested in the comments above?